ExtractData/PKHeX.ExtractData/ExtTMHM3.cs

```diff
@@ -0,0 +1,376 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PKHeX.ExtractData
+{
+    class ExtTMHM3 : ExtMoveTutors4
+    {
+        internal override GenerationData GData => Generation3Data.Create();
+
+        static readonly string[] TMMoves_Gen4 =
```

Comment:
TMMoves_Gen4 was used to find the move tutors offset in the rom. Most of the code to find offset was deleted after i have the offsets because is not needed anymore.